packages/acceptance-tests/pkg-tests-specs/sources/commands/unlink.test.js

```diff
@@ -59,22 +59,22 @@ describe(`Commands`, () => {
             cwd: `${path}/packages/workspace`,
           });
 
-          await expect(readJson(`${path}/package.json`)).resolves.toMatchObject({
+          await expect(readJson(`${path}/package.json`)).resolves.toEqual(expect.objectContaining({
             resolutions: {
               [`my-package-a`]: `portal:${npath.toPortablePath(`${tmp}/my-package-a`)}`,
               [`my-package-b`]: `portal:${npath.toPortablePath(`${tmp}/my-package-b`)}`,
             },
-          });
+          }));
 
           await run(`unlink`, `${tmp}/my-package-b`, {
             cwd: `${path}/packages/workspace`,
           });
 
-          await expect(readJson(`${path}/package.json`)).resolves.toMatchObject({
+          await expect(readJson(`${path}/package.json`)).resolves.toEqual(expect.objectContaining({
```

Comment:
toMatchObject passes on partial matches, which means the test wasn't actually doing a proper assertion